packages/atomic/src/themes/default.css

```diff
@@ -1,19 +1,19 @@
 :root {
```

Comment:
prettier did not run on master. I'm doing a fix for that in another PR, which in turn will mean these code is valid